package-support-policy.md

```diff
@@ -0,0 +1,90 @@
+# Support Policy for .NET Packages
+
+NuGet packages owned by the .NET team are supported as long as they target at least one framework that is still actively supported.
+
+The minimum supported .NET versions are:
```

Comment:
Is it necessary to include this information. We don't want more places to update.

Reply:
Do we have a page where we indicate what are the minimum supported .NET versions? I could link to that instead.

Reply:
- https://github.com/dotnet/core/blob/main/releases.md
- https://dotnet.microsoft.com/platform/support/policy